src/processes.c

```diff
@@ -2029,6 +2029,9 @@ static int ps_read (void)
 	int stopped  = 0;
 	int paging   = 0;
 	int blocked  = 0;
+	/* returns the number of processes with a command line argument */
```

Comment:
Another nit: `processes with a command line`, not `processes with a command line argument`. Also below.

Reply:
Done.